docs/part1.md

```diff
@@ -572,12 +569,16 @@ Jumping between side panes is a bit of a hassle, so instead we will speed up our
 
 This is our nascent dashboard.
 It will speed up our data exploration tremendously, but first we have to build it out a bit.
-**Add to your dashboard** at least:
+**Add to your dashboard**:
 
 - the periodic table ("Elements / Formula" > "Elements")
 - "Elements / Formula" > "Number of Elements"
 - "Author / Origin / Dataset" > "Dataset Name"
-- ~~"Electronic" > "Electronic Properties"~~ is not necessary, since there are only DOS
+
+<!-- And **remove from the dashboard**:
+
+- "Electronic" > "Electronic Properties" - not necessary, since there are only DOS
+@ndaelman-hu I didn't understand this, you had it striked out so I thought you wanted to remove it, but I guess you were just saying it was not necessary to add?? -->
```

Comment:
Exactly, it was following the filter list from scenario 2 (since scenario 3 jumps off from there), but with the DOS left out.
I could either be more explicit and say the list of filters in scenario 2 (enumerating them) and leaving the DOS striked out, or we just remove it.

Reply:
I think just leave it out. At least for me, some of the references back to Scenario 2 were not so obvious.